finality-aleph/src/network/manager/discovery.rs

```diff
@@ -259,12 +140,10 @@ impl Discovery {
             AuthenticationBroadcast(authentication) => {
                 self.handle_broadcast(authentication, handler)
             }
-            Request(node_ids, authentication) => {
-                self.handle_request(node_ids, authentication, handler)
-            }
-            Authentications(authentications) => {
-                (self.handle_response(authentications, handler), Vec::new())
-            }
+            Authentication(authentication) => (
```

Comment:
It took me 10 secs to realize that we're returning tuple from here, maybe below formatting is more readable?
``` rust
 match message {
            AuthenticationBroadcast(authentication) => {
                self.handle_broadcast(authentication, handler)
            }
            Authentication(authentication) =>
                (self.handle_authentication(authentication, handler), Vec::new()),
           }
}
```

Reply:
Done